### DISCUSSION TITLE
Running this on Docker Synology Illegal instruction (core dumped)

### DISCUSSION THREAD

**Discussion by figarospares** [aahnik/tgcf#425] [Q&A]
HI I have been trying to get this working on Synology RS812+ and it keeps restarting when I check the log all I get is 
/venv/lib/python3.10/site-packages/tgcf/web_ui
Illegal instruction (core dumped)

I have tried different things but still the same.

But when I tried on docker desktop works perfect with out any issues

Any ideas would be very helpful as don't want to run on desktop all the time

**Comment by JoshYang3** [comment]
I met the same error as you did, the only difference is I am working on Oracle Cloud Instance and I fixed it by running

1.  `git clone https://github.com/aahnik/tgcf.git`
2.  `cd tgcf` (Or where the Dockerfile is located, can use `find . -type f -name Dockerfile` to find it)
3.  `docker build -t tgcf .`
4.  `docker run -d -p 8501:8501 --env-file .env --name tgcf tgcf`

And it should be running perfectly.
